.travis.yml

```diff
@@ -5,19 +5,31 @@ php:
   - 5.4
   - 5.5
   - 5.6
+  - 7.0
   - hhvm
- 
-matrix:
-    allow_failures:
-        - php: hhvm
 
+sudo: false
+
+cache:
+    directories:
+        - $HOME/.composer/cache
+
+matrix:
+    fast_finish: true
+    include:
+        - php: 5.3
+          env: COMPOSER_FLAGS="--prefer-lowest" SYMFONY_DEPRECATIONS_HELPER=weak
+        - php: 7.0
+          env: SYMFONY_VERSION=2.7.*
+        - php: 7.0
+          env: SYMFONY_VERSION=2.8.* COVERAGE=true
 
 before_script:
     - composer self-update
-    - composer install --dev --prefer-source
+    - if [ "$SYMFONY_VERSION" != "" ]; then composer require --dev --no-update symfony/symfony=$SYMFONY_VERSION; fi
+    - composer update $COMPOSER_FLAGS --prefer-source
 
-script: phpunit --coverage-clover clover
+script: if [ "$COVERAGE" == "true" ]; then phpunit --coverage-clover=clover; else phpunit; fi
```

Comment:
that's why the older builds end up with a parse error: https://github.com/travis-ci/travis-ci/issues/5206